src/xturing/engines/gptj_utils/gptj.py

```diff
@@ -1,10 +1,21 @@
+from typing import Optional, Tuple, Union
+
 import torch
 import torch.nn as nn
-from typing import Optional, Union, Tuple
-from transformers.models.gptj.modeling_gptj import (
-    apply_rotary_pos_emb,
-    fixed_pos_embedding,
-)
+from transformers.models.gptj.modeling_gptj import apply_rotary_pos_emb
+
+
+def fixed_pos_embedding(x, seq_dim=1, seq_len=None):
```

Comment:
@StochasticRomanAgeev where did you find this fix? Is this a known issue?

Reply:
No, just went through src's of transformers and found that this function is missing